airflow/configuration.py

```diff
@@ -580,10 +589,24 @@ def getint(self, section, key):
     def getfloat(self, section, key):
         return float(self.get(section, key))
 
+    def remove_option(self, section, option):
+        try:
+            ConfigParser.remove_option(self, section, option)
+        except ConfigParser.NoSectionError:
+            pass
+        try:
+            ConfigParser.remove_option(self.defaults, section, option)
+        except ConfigParser.NoSectionError:
+            pass
+
     def read(self, filenames):
         ConfigParser.read(self, filenames)
         self._validate()
 
+    def has_option(self, section, option):
```

Comment:
It's tricky semantically; how do we handle:

```python
if conf.has_option('something'):
    conf.remove_option('something')
```

if the config has `something_cmd` but not `something`?

If someone is trying to get a config value, they should just use `get`.

I don't see a great answer.